etc/otel-jmx.config.yaml

```diff
@@ -95,13 +95,13 @@ rules:
        metric: responsive.kafka.streams.flush.latency.max
        desc: the maximum time it took to flush the commit buffer
        unit: '{milliseconds}'
-     flush-fenced-rate:
-       metric: responsive.kafka.streams.flush.fenced.rate
-       desc: the rate of commit buffer flushes that were fenced
+     flush-errors-rate:
```

Comment:
yeah it would be nice to have a test that fails if our metrics get out of sync but I have no idea how to do that lol

Maybe as a GHA in the PR builder workflow?